modules/custom/d_demo/blocks/after_footer_menu.yml

```diff
@@ -0,0 +1,11 @@
+info:
+  type: block_plugin
+  plugin: 'system_menu_block:after_footer_menu'
+  langcode: en
+  region: footer_main
+  weight: 0
+  settings:
+    status: true
+    provider: system_menu_block
+    label: 'After footer menu'
```

Comment:
After footer menu,czyli taki Bottom footer ?

Reply:
tylko pytam, bo w sumie nie do znam contekstu

Reply:
Bottom footer by lepiej pasowało, bo jak ktoś nie ma footerów w kolumnach to nie zrozumie o co chodzi